samples/kvsWebRTCClientMaster.c

```diff
@@ -199,6 +199,12 @@ PVOID sendVideoPackets(PVOID args)
         goto CleanUp;
     }
 
+    MUTEX_LOCK(pSampleConfiguration->sampleConfigurationObjLock);
```

Comment:
This fix doesn't reduce the startup latencies for viewers other than the first. To make this work for other viewers, we need to buffer all of the frames from the latest keyframe to the current frame. This will add complexities to the sample, and, thus, it's explicitly ignored for the sake of an example.

Reply:
This would be a separate feature by itself. Similar to pre-buffering work for the producer SDK. We can add a work item for the samples to pre-buffer from key-frame